templates/indieauth-authenticate-form.php

```diff
@@ -13,7 +13,8 @@
 			printf(
 				'<p>' . __( 'The app <strong>%1$s</strong> would like to sign you in as <strong>%2$s</strong>.', 'indieauth' ) . '</p>',
 				$client_id,
-				$me
+				get_url_from_user( $current_user->ID )
```

Comment:
there's a very very small possibility of the return null value coming through and it's not being handled here.  There should be a better textual representation of null in this situation.

Reply:
Is there really a chance? If the user is not logged in, he never see this site.

Reply:
it takes a minute to defend against it. imo the fact the function is capable of returning null means it should get handled. I don't think you can guarantee the state of current_user, as other plugins CAN mess it up.